avocado/plugins/base.py

```diff
@@ -63,13 +61,9 @@ class CLICmd(Plugin):
     Plugins that want to add extensions to the run command should use the
     'avocado.plugins.cli.cmd' namespace.
     """
-    __metaclass__ = abc.ABCMeta
     name = None
     description = None
 
-    def __init__(self):
-        super(CLICmd, self).__init__()
-
     def configure(self, parser):
```

Comment:
I'm looking at the examples on https://docs.python.org/2/library/abc.html and they use only the __metaclass__  for class and then `@abstractmethod` for mandatory metods. So I think it's correct.